test/functional/wallet_basic.py

```diff
@@ -22,6 +22,8 @@ class WalletTest(BitcoinTestFramework):
     def set_test_params(self):
         self.num_nodes = 4
         self.setup_clean_chain = True
+        # node 2 needs a txindex to look up raw transactions
+        self.extra_args = [[], [], ["-txindex"], []]
```

Comment:
This is part of the wrong commit. The `sendtoaddress` test only calls `getrawtransaction` on a transaction in the mempool, which doesn't need a tx index.

Also, the `sendmany` test that relies on this *could* just pass the block hash explicitly to avoid building the tx index.

Reply:
Actually, you already removed the `getrawtransaction` from sendmany, so this seems completely unnecessary?

Reply:
Didn't realize that, thanks. Removed these 2 lines!